### PR TITLE
Issue #3843: Disallow import of interfaces from java.util.stream due to coverage problem

### PR DESCRIPTION
**Pull request by MEZk**
#3843 

I did not use ForbitCertainImports as it cannot exclude test classes where usage of java.util.stream should be allowed.

### CODE REVIEW

File: config/import-control.xml [hunk 1]
```diff
@@ -16,6 +16,18 @@
   <allow pkg="org.apache.commons.logging"/>
   <allow pkg="org.xml.sax"/>
 
+  <!--Disallowed until https://github.com/mojohaus/cobertura-maven-plugin/issues/29-->
+  <disallow class="java.util.stream.Stream"/>
```
Comment (rnveach) [inline config/import-control.xml]: Why don't we do `<disallow pkg="java.util.stream">` ?

Comment (romani) [inline config/import-control.xml]: @MEZk , as you claim it is related to all other Stream classes in this package "java.util.stream", lets disallow all of them (not that much of them in package)

**Comment by romani** [comment]
>I did not use ForbitCertainImports as it cannot exclude test classes where usage of java.util.stream should be allowed

Can we suppress whole check in tests ? We could give configuration certain id and suppress by that id.

File: config/import-control.xml (hunk 1 above)
Comment (MEZk) [inline config/import-control.xml]: No, it is related only to interfaces in the package.

**Comment by MEZk** [comment]
@romani 
> Issue #3843: Disallow import of interfaces from java.util.stream due to coverage problem

Done.

**Comment by romani** [comment]
I am put this PR on hold to let us test another PR #3853 that will catch missed coverage on whole file.
As soon as #3853 fail on Travis, we could merge this PR.

**Pull request merged**